Octokit.Tests/Clients/SearchClientTests.cs

```diff
@@ -209,69 +215,89 @@ public void TestingTheLanguageQualifier()
                     Arg.Is<Dictionary<string, string>>(d => d["q"] == "github+language:Ruby"));
             }
 
-            [Fact]
-            public void TestingTheCreatedQualifier_GreaterThan()
+            [Theory]
+            [InlineData(DateTimeStrategies.DateTimeStrategy)]
+            [InlineData(DateTimeStrategies.DateTimeOffSetStrategy)]
+            public void TestingTheCreatedQualifier_GreaterThan(DateTimeStrategies strategy)
             {
                 var connection = Substitute.For<IApiConnection>();
                 var client = new SearchClient(connection);
                 var request = new SearchUsersRequest("github");
-                request.Created = DateRange.GreaterThan(new DateTime(2014, 1, 1));
-                client.SearchUsers(request);
+                request.Created = strategy == DateTimeStrategies.DateTimeStrategy 
+                    ? DateRange.GreaterThan(new DateTime(2014, 1, 1)) 
+                    : DateRange.GreaterThan(new DateTimeOffset(2014, 1, 1, 2, 4, 6, new TimeSpan(10, 0, 0)));
+                client.SearchUsers(request);        
                 connection.Received().Get<SearchUsersResult>(
                     Arg.Is<Uri>(u => u.ToString() == "search/users"),
-                    Arg.Is<Dictionary<string, string>>(d => d["q"] == "github+created:>2014-01-01"));
+                    Arg.Is<Dictionary<string, string>>(d => d["q"] == $"github+created:{request.Created}"));
```

Comment:
I'd prefer the unit test actually tests the formatted string eg '2014-1-1'T'020406 1000' (or whatever it should be)

Otherwise these tests are susceptible to whatever formatting bug may exist inside `DateRange` class.  

EG Im pretty sure the way you've implemented it at the moment isn't correctly formatting the string, yet all these tests pass (because you are using the `DateRange` class to test the `DateRange` class! 🤣

Reply:
Hi @ryangribble 
Has taken me a while to make these changes due to other commitments, however, they're now there.

Let me know what you think.